download/download_test.go

```diff
@@ -0,0 +1,42 @@
+package download
+
+import (
+	"testing"
+
+	"github.com/alexandrasp/taskcluster-cli/extpoints"
+)
+
+func TestSummary(t *testing.T) {
+	dl := download{}
+	expectedStr := "Download an artifact"
+	result := dl.Summary()
+	if result != expectedStr {
+		t.Fatalf("Expected %s got %s", expectedStr, result)
+	}
+}
+
+func TestUsage(t *testing.T) {
+	dl := download{}
+	result := dl.Usage()
+	t.Log(result)
+}
+
+func TestInit(t *testing.T) {
+	dl := download{}
+	result := dl.ConfigOptions()
+	t.Log(result)
+}
+
+func TestExecute(t *testing.T) {
+	dl := download{}
+	myContext := extpoints.Context{}
+	myContext.Arguments = make(map[string]interface{})
+	myContext.Arguments["download"] = "download"
+	myContext.Arguments["<taskId>"] = "JCwcT6cUQ666JJKR4gpWVw"
+	myContext.Arguments["<runId>"] = ""
+	myContext.Arguments["<artifact>"] = "public/test_info//resource-usage.json"
+
+	response := dl.Execute(myContext)
```

Comment:
You are executing but not actually checking the result here. @jonasfj can we use the same S3 tests url to upload a dummy artifact for tests?